website/docs/docs/explore/access-dbt-insights.md

```diff
@@ -152,6 +152,15 @@ To access <Constant name="explorer" />, click on the **<Constant name="explorer"
 
 <Lightbox src="/img/docs/dbt-insights/insights-explorer.png" width="90%" title="dbt Insights integrated with dbt Explorer" />
 
+### Set Jinja environment
```

Comment:
Can we now remove the "Coming soon" later on the page [here](https://github.com/dbt-labs/docs.getdbt.com/pull/7504/files#diff-8aa0be69121de88daf344ed139a3dcfe214531cb172855a3344b62ab38bde6aaR178)?

Reply:
Good catch! Removed